### PR TITLE
ffmpeg: update to 4.2.2-Matrix-Alpha1

### PR DESCRIPTION
**Pull request by lrusak**
This is to go along with https://github.com/xbmc/xbmc/pull/17300

Only updated the version and the patch that I maintain.

Will need the following updates:
 - [x] RPi patches
 - [x] MVC patches
 - [x] V4L2 Request patches
 - [x] V4L2 drmprime patches

### CODE REVIEW

**Comment by lrusak** [comment]
tested on linux-gbm with vaapi and all seems fine. http://ix.io/28LC

**Comment by Kwiboo** [comment]
please pick/squash/fixup https://github.com/Kwiboo/LibreELEC.tv/commit/c4a9a8c5f19ccf58c564e63d552e81ce74dbbcf7 for updated V4L2 Request patches

**Comment by MilhouseVH** [comment]
Generic test build with `ffmpeg-4.2.2-Matrix-Alpha1`: http://milhouse.libreelec.tv/builds/testing/Generic/LibreELEC-Generic.x86_64-9.80-Milhouse-20200131051644-%230131-gbe02e12.tar

Log: http://ix.io/28Me

```
2020-01-31 08:28:26.115 T:1161  NOTICE: Starting Kodi (19.0-ALPHA1 Git:177edb5). Platform: Linux x86 64-bit
2020-01-31 08:28:26.115 T:1161  NOTICE: Using Release Kodi x64 build
2020-01-31 08:28:26.115 T:1161  NOTICE: Kodi compiled 2020-01-31 by GCC 9.2.0 for Linux x86 64-bit version 5.4.16 (328720)
2020-01-31 08:28:26.115 T:1161  NOTICE: Running on LibreELEC (Milhouse.testing): devel-20200131051644-#0131-gbe02e12 [Build #0131] 9.80, kernel: Linux x86 64-bit version 5.4.16
2020-01-31 08:28:26.115 T:1161  NOTICE: FFmpeg version/source: 4.2.2-Kodi
2020-01-31 08:28:26.115 T:1161  NOTICE: Host CPU: Intel(R) Core(TM) i5-6260U CPU @ 1.80GHz, 4 cores available
```

**Comment by lrusak** [comment]
@Kwiboo thanks for the update.

File: packages/multimedia/ffmpeg/package.mk [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 PKG_NAME="ffmpeg"
 # Current branch is: release/4.0-kodi
-PKG_VERSION="4.0.4-Leia-18.4"
-PKG_SHA256="e11e7594af35f36ab2711252c3d6bb106908f26605498aef4a9be2d7bc001db2"
+PKG_VERSION="2c6b3f357331e203ad87214984661c40704aceb7"
```
Comment (MilhouseVH) [inline packages/multimedia/ffmpeg/package.mk]: Can this be updated to 4.2.2-Matrix-Alpha1?

**Comment by lrusak** [comment]
updated and squashed, should be good to go

**Pull request merged**